extensions/security-webauthn/runtime/src/main/java/io/quarkus/security/webauthn/WebAuthnRunTimeConfig.java

```diff
@@ -24,6 +23,172 @@
 @ConfigRoot(phase = ConfigPhase.RUN_TIME)
 public interface WebAuthnRunTimeConfig {
 
+    /**
+     * PublicKeyCredential
+     * https://www.iana.org/assignments/cose/cose.xhtml#algorithms
+     */
+    public enum PublicKeyCredential {
```

Comment:
WebAuthn standard uses the name `PublicKeyCredential` for [other type](https://w3c.github.io/webauthn/#publickeycredential).
COSE algorithm type should be `COSEAlgorithmIdentifier`.
https://www.w3.org/TR/webauthn-3/#typedefdef-cosealgorithmidentifier

Reply:
Absolutely, that's a mistake that I've no idea how it ended up there :-/

Reply:
Ah, probably because the config parameter is named `pubKeyCredParams`, which, BTW I should change to a non-abbreviated name.